tmail-backend/imap-extensions/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3.8'
+
+networks:
+  emaily:
+    driver: bridge
+
+services:
+  tmail-backend:
+    image: linagora/tmail-backend-memory:latest
+    container_name: tmail-backend
+    hostname: tmail-backend.local
+    ports:
+      - "80:80"
+      - "25:25"
+      - "110:110"
+      - "143:143"
+      - "465:465"
+      - "587:587"
+      - "993:993"
+      - "8000:8000"
+    networks:
+      - emaily
+    volumes:
+      - $PWD/jwt_publickey:/root/conf/jwt_publickey
+      - $PWD/jwt_privatekey:/root/conf/jwt_privatekey
+      - $PWD/src/main/conf/smtpserver.xml:/root/conf/smtpserver.xml
+      - $PWD/src/main/conf/imapserver.xml:/root/conf/imapserver.xml
+      - ./target/imap-extensions-1.0.0-SNAPSHOT-jar-with-dependencies.jar:/root/extensions-jars/imap-extensions.jar
```

Comment:
Please add the dependency to apps so that it end up in the class path without the external-jars trick?